doc/source/whatsnew/v1.0.3.rst

```diff
@@ -23,6 +23,7 @@ Fixed regressions
 Bug fixes
 ~~~~~~~~~
 
+
```

Comment:
Can you revert unrelated changes? Looks like blank space and file permissions were changed here

Reply:
@DataInformer can you address this

Reply:
can you do this.

Reply:
Sorry, I thought I did this before, but apparently it reverted.  Should be undone again now.